Sources/BraintreeCore/BTClientMetadata.swift

```diff
@@ -11,56 +13,45 @@ import Foundation
 /// the sessionID should remain constant. To achieve this, users of this class
 /// should use `mutableCopy` to create a new copy based on the existing session
 /// and then update the object as needed.
-@objcMembers public class BTClientMetadata: NSObject, NSMutableCopying {
-    
+@_documentation(visibility: private)
+@objcMembers public class BTClientMetadata: NSObject, NSCopying {
+
+    /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
     /// Integration type
+    @_documentation(visibility: private)
     public var integration: BTClientMetadataIntegration
-    
+
+    /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
     /// Integration source
+    @_documentation(visibility: private)
     public var source: BTClientMetadataSource
-    
+
+    /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
     /// Auto-generated UUID
+    @_documentation(visibility: private)
     public var sessionID: String
     
-    /// String representation of the integration
-    public var integrationString: String {
-        integration.stringValue
-    }
-    
-    /// String representation of the source
-    public var sourceString: String {
-        source.stringValue
-    }
```

Comment:
Since these were public anyways, it made sense to remove them and expose the `stringValue` directly.